pages/data-providers/[data-provider-id]/deposit-dates.jsx

```diff
@@ -119,7 +119,12 @@ const DepositDates = ({
         <p>
           The chart displays work distribution per deposit time lag in days.
         </p>
-        <TimeLagChart data={store.depositDates.timeLagData} />
+        {Boolean(store.depositDates.timeLagData.length) && (
```

Comment:
It's better to write `length > 0`. Check Airbnb's JavaScript Style Guide [22.6](https://github.com/airbnb/javascript#coercion--booleans) and [15.3](https://github.com/airbnb/javascript#comparison--shortcuts)